practice-areas/engineering/drupal/drupal-contrib-first-module-development.md

```diff
@@ -0,0 +1,15 @@
+## Drupal Contrib First module developement
```

Comment:
I think you don't need to duplicate the header on this page. I believe the solution to any linting errors is to add the frontmatter title. Example:

```
---
title: Drupal Contrib First module development
---
```

I also noticed you want to do that same in the 'contrib-first' page as well. Also small typo in the title.

Reply:
Or the solution is to use single '#' for page title h1 and not two '#'

Reply:
Double title issue seen here https://civicactions-handbook--1451.org.readthedocs.build/en/1451/practice-areas/engineering/drupal/drupal-contrib-first-module-development/

Reply:
Cool, still seeing an extra header so I think that extra one can be dropped.

Reply:
late push fixed it :)